docs/guides/qiskit-addons-cutting-gates.ipynb

```diff
@@ -0,0 +1,470 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "682d5724-d8bc-40e6-b42e-9224a600fe7a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Reconstructed expectation value: 0.50952148\n",
+      "Exact expectation value: 0.50497603\n",
+      "Error in estimation: 0.00454545\n",
+      "Relative error in estimation: 0.00900132\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Generate the subexperiments and sampling coefficients\n",
+    "subexperiments, coefficients = generate_cutting_experiments(\n",
+    "    circuits=qpd_circuit, observables=observable.paulis, num_samples=np.inf\n",
+    ")\n",
+    "\n",
+    "# Set a backend to use and transpile the subexperiments\n",
+    "backend = FakeManilaV2()\n",
+    "pass_manager = generate_preset_pass_manager(\n",
+    "    optimization_level=1, backend=backend\n",
+    ")\n",
+    "isa_subexperiments = pass_manager.run(subexperiments)\n",
+    "\n",
+    "# Set up the Qiskit Runtime Sampler primitive, submit the subexperiments, and retrieve the results\n",
+    "sampler = SamplerV2(backend)\n",
+    "job = sampler.run(isa_subexperiments)\n",
+    "results = job.result()\n",
+    "\n",
+    "\n",
+    "# Reconstruct the results\n",
+    "reconstructed_expval_terms = reconstruct_expectation_values(\n",
+    "    results,\n",
+    "    coefficients,\n",
+    "    observable.paulis,\n",
+    ")\n",
+    "\n",
+    "# Apply the coefficients of the original observable\n",
+    "reconstructed_expval = np.dot(reconstructed_expval_terms, observable.coeffs)\n",
+    "\n",
+    "estimator = EstimatorV2()\n",
+    "exact_expval = (\n",
+    "    estimator.run([(circuit, observable, [0.4] * len(circuit.parameters))])\n",
+    "    .result()[0]\n",
+    "    .data.evs\n",
+    ")\n",
+    "print(\n",
+    "    f\"Reconstructed expectation value: {np.real(np.round(reconstructed_expval, 8))}\"\n",
+    ")\n",
+    "print(f\"Exact expectation value: {np.round(exact_expval, 8)}\")\n",
+    "print(\n",
+    "    f\"Error in estimation: {np.real(np.round(reconstructed_expval-exact_expval, 8))}\"\n",
+    ")\n",
+    "print(\n",
+    "    f\"Relative error in estimation: {np.real(np.round((reconstructed_expval-exact_expval) / exact_expval, 8))}\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "64f44348-3ad1-4bf4-9a3a-cf0d73ca3431",
+   "metadata": {},
+   "source": [
+    "<Admonition type=\"caution\" title=\"Note about observable coefficients\">\n",
+    "    To accurately reconstruct the expectation value, the coefficients of the original observable (which are different from the coefficients in the output of `generate_cutting_experiments()`) must be applied to the output of the reconstruction since this information was lost when the cutting experiments were generated or when the observable was expanded.\n",
```

Comment:
```suggestion
    "    To accurately reconstruct the expectation value, the coefficients of the original observable (which are different from the coefficients in the output of `generate_cutting_experiments()`) must be applied to the output of the reconstruction, since this information was lost when the cutting experiments were generated or when the observable was expanded.\n",
```